src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Data/EmployerAccountRepository.cs

```diff
@@ -8,21 +8,25 @@
 using System.Collections.Generic;
 using System.Data;
 using System.Data.Entity;
+using System.Diagnostics;
 using System.Linq;
 using System.Threading.Tasks;
 using SFA.DAS.Authorization;
 using SFA.DAS.EAS.Domain.Models.EmployerAgreement;
+using Stopwatch = System.Diagnostics.Stopwatch;
 
 namespace SFA.DAS.EAS.Infrastructure.Data
 {
     public class EmployerAccountRepository : BaseRepository, IEmployerAccountRepository
     {
         private readonly Lazy<EmployerAccountsDbContext> _db;
+        private readonly ILog _logger;
 
         public EmployerAccountRepository(EmployerApprenticeshipsServiceConfiguration configuration, ILog logger, Lazy<EmployerAccountsDbContext> db)
             : base(configuration.DatabaseConnectionString, logger)
         {
             _db = db;
+            _logger = logger;
```

Comment:
This is already done in base class, it doesn't need to be done here too (but no problem if it remains as it is more explicit doing it here)